chainweb.cabal

```diff
@@ -95,7 +97,7 @@ common warning-flags
 custom-setup
     setup-depends:
           Cabal
-        , base >= 4.11 && < 5
+        , base >= 4.12 && < 5
```

Comment:
Why are we bumping base here?

Reply:
It establishes GHC 8.6 officially as our lower bound. It also makes sure `-funclutter-valid-hole-fits` works for anybody who compiles this (it doesn't exist for <= 8.4)